pom.xml

```diff
@@ -48,6 +48,8 @@
 
   <properties>
     <stack.version>4.0.0-SNAPSHOT</stack.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
CC @vietj to make sure you are OK with this change

Reply:
Why do we need this as it's already defined in https://github.com/vert-x3/vertx-parent/blob/master/pom.xml?

Reply:
I thought I ran into an incompatible class version change and this fixed it, but I just double checked and it looks like we were indeed compiling to 1.8 before, so I must have been mistaken -- I'll remove this